var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -47,14 +48,18 @@ class Cp2k(Package):
     depends_on('lapack')
     depends_on('blas')
     depends_on('fftw')
-    depends_on('libint@:1.2', when='@3.0')
+    depends_on('libint@:1.2', when='@3.0,4.1')
 
     depends_on('mpi', when='+mpi')
     depends_on('scalapack', when='+mpi')
     depends_on('plumed+shared+mpi', when='+plumed+mpi')
     depends_on('plumed+shared~mpi', when='+plumed~mpi')
-    depends_on('pexsi', when='+mpi')
-    depends_on('wannier90', when='+mpi')
+    depends_on('pexsi+fortran', when='+mpi')
+
+    # Apparently cp2k@4.1 needs an "experimental" version of libwannier.a
+    # which is only available contacting the developer directly. See INSTALL
+    # in the stage of cp2k@4.1
```

Comment:
No, it builds. The comment is just to explain why the dependency on `wannier90` is only for `cp2k@3.0`, while in principle also `cp2k@4.1` can be built with the optional `wannier90` support. 

The long story is that I got a symbol missing at linking time (`wannier_setup_simple`) and I had to inspect `cp2k` and `wannier90` sources to understand that `cp2k@4.1` was calling a function that was not in the `libwannier.a` library. Then I started looking into every file in `cp2k` and found the message:

> ==== 2p. libwannier90 (optional, interface to Wannier90 code) ====
>  * Needs an experimental version of wannier90 (currently only available from
>    hutter@chem.uzh.ch)
>  * Add -D__WANNIER90 to DFLAGS, -L$(LIBWANNIER90_DIR)-lwannier90 to LIBS.

hence the comment above.